docs/plugins.rst

```diff
@@ -0,0 +1,131 @@
+specifying the provider STORET.
+
+**Data Usage** See citation information at `WQP User Guide <https://www.waterqualitydata.us/portal_userguide/>`_. Requests to the WQP web services are logged. See :doc:`/quick_guide`.
+
+Section 1: EPA Data Source Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+None.
+Data are publicly available and accessed via the `Water Quality Portal Web Services <https://www.waterqualitydata.us/webservices_documentation/>`_.
+Please follow data usage guidelines at `User Guide <https://www.waterqualitydata.us/portal_userguide/>`_ .
+
+
+Section 2: Using the EPA plugin in BASIN-3D
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Install basin3d using the instructions in the :doc:`/getting_started`.
+
+2. Import the EPA WQX plugin:
+
+.. code-block::
+
+    >>> from basin3d.plugins import epa
+
+3. Register the plugin (and any other you have imported):
+
+.. code-block::
+
+    >>> synthesizer = synthesis.register()
+
+4. Find the available monitoring feature IDs (aka location identifiers). Note: These monitoring feature IDs are required parameters for BASIN-3D data requests
+
+Option 1: Specify a `USGS HUC <https://water.usgs.gov/GIS/huc.html>`_ 2, 4, 6, or 8-digit code in the parent_feature argument. Prefix the HUC identifier with "EPA-".
+The BASIN-3D USGS plugin can be used to find USGS HUC information and identifiers:
+
+.. code-block::
+
+    >>> monitoring_features = synthesizer.monitoring_features(parent_feature=['EPA-14020001'])
+    >>> for monitoring_feature in monitoring_features:
+    >>>     print(f'{monitoring_feature.id} --- {monitoring_feature.name}')
+    EPA-21COL001-000058 --- TAYLOR RIVER AT ALMONT
+    EPA-21COL001-000078 --- EAST RIVER AT CONFL. WITH TAYLOR
+    EPA-21COL001-000150 --- SLATE RIVER ABOVE COAL CREEK
+    ...
+
+Option 2: Specify one or more EPA Site IDs, prefixed by "EPA-":
+
+.. code-block::
+
+    >>> monitoring_features = synthesizer.monitoring_features(parent_feature=['EPA-CORIVWCH_WQX-176', 'EPA-11NPSWRD_WQX-BLCA_09128000'])
+    >>> for monitoring_feature in monitoring_features:
+    >>>     print(f'{monitoring_feature.id} --- {monitoring_feature.name}')
+    EPA-11NPSWRD_WQX-BLCA_09128000 --- GUNNISON RIVER BELOW GUNNISON TUNNEL, CO
+    EPA-CORIVWCH_WQX-176 --- Uncompahgre -  Confluence Park
+
+
+5. Request time series data. Query argument aggregation_duration supports "NONE" or "DAY". See `epa_mapping.py <https://github.com/BASIN-3D/basin3d/blob/main/basin3d/plugins/epa_mapping.py>`_:
+
+.. code-block::
+
+    >>> measurement_timeseries_tvp_observations = synthesizer.measurement_timeseries_tvp_observations(monitoring_feature=['EPA-CCWC-COAL-26', 'EPA-CCWC-MM-29 WASH #3'], observed_property=['As', 'WT', 'DO'], start_date='2010-01-01', end_date='2011-01-01', aggregation_duration='NONE')
+
+6. Synthesized data should be cited following the Water Quality Portal data use policies.
```

Comment:
Is there supposed tbe a citation here?

Reply:
No. I changed the text to reference the Data Usage detail in the intro section. See if works.